test/common/validation.go

```diff
@@ -92,14 +94,39 @@ func (client *Client) FindAnyLogContents(podName string, contents []string) (boo
 	if err != nil {
 		return false, err
 	}
+	eventContentsSet, err := parseEventContentsFromPodLogs(string(logs))
+	if err != nil {
+		return false, err
+	}
 	for _, content := range contents {
-		if strings.Contains(string(logs), content) {
+		if _, ok := eventContentsSet[content]; ok {
 			return true, nil
 		}
 	}
 	return false, nil
 }
 
+//Example log entry: 2019/08/21 22:46:38 {"msg":"Body-type1-source1--extname1-extval1-extname2-extvalue2","sequence":"1"}
```

Comment:
```suggestion
// Example log entry: 2019/08/21 22:46:38 {"msg":"Body-type1-source1--extname1-extval1-extname2-extvalue2","sequence":"1"}
```

Add a single space after `//`. This line and the following ones.